reporter/src/funTest/assets/NPM-is-windows-1.0.2-expected-NOTICE

```diff
@@ -1,4 +1,5 @@
-This project contains or depends on third-party software components pursuant to the following licenses:
+This project contains or depends on third-party software components.
```

Comment:
Ideally, the code changes that cause the expected test results for the legacy notice reporter to change would have been done in a separate commit, so that this commit really only add the implementation of the new notice reporter. Maybe these changes could have even been squash into the change that separates out the abstract notice reporter.

Reply:
But *why* does this wording *have* to change after all?